analyzers/rspec/vbnet/S5693_vb.net.html

```diff
@@ -0,0 +1,28 @@
+<p>Rejecting requests with significant content length is a good practice to control the network traffic intensity and thus resource consumption in
```

Comment:
The rule was specified for C# but not yet for VB.NET. We need to update the jira rspec by adding a subtask for VB with code examples.